docs/handbook/community.md

```diff
@@ -34,6 +34,100 @@ Link to PDF Document: <a href= "../pdf/HowToInstallOracleDB.pdf" target="_blank"
 <p>Unfortunately, it is not possible to run Oracle Database version 12 and above on OpenIndiana.</p>
 </div>
 
+# Hipster Handbook - Squeak Smalltalk-80
+
+The following notes document the steps to install Squeak on OpenIndiana Hipster, and how to use this implementation of Smalltalk.  For further information, feedback or suggestions, please contact me (David Stes).
+
+Smalltalk is a graphical development environment and programming language.  A famous reference is the 700-page book : "Smalltalk-80: The Language and its Implementation"
+
+Adele Goldberg and David Robson
+Xerox Palo Alto Research Center
+Addison-Wesley Publishing Company, Reading,
+Massachussetts, Menlo Park, California, 1983, ISBN 0-201-11371-6
+
+There also exist more recent books on Squeak.
+
+See the <a href="http://www.squeak.org" target="_blank">Squeak website</a> for more information, specifically the Documentation section.
+
+Also see the Squeak Wiki for more information : <a href= "http://wiki.squeak.org" target="_blank">Squeak Wiki</a>
+
+## How to install Squeak Smalltalk-80 on OpenIndiana
+
+<i class="fa fa-info-circle fa-lg" aria-hidden="true"></i>OpenIndiana Prerequisites
+<div class="well">
+<p>When installing OpenIndiana, you will need a desktop (graphical) environment, such as the MATE desktop.
```

Comment:
The part about MATE desktop should be moved to the general installation if not already present and a link to the relevant section. Just to avoid duplicate information.